src/lib/markbind/src/lib/markdown-it/index.js

```diff
@@ -45,17 +45,44 @@ markdownIt.renderer.rules.fence = (tokens, idx, options, env, slf) => {
   if (!highlighted) {
     str = markdownIt.utils.escapeHtml(str);
   }
+  let highlightLines = token.attrGet('highlight-lines')
+  let lineNumbers = []
+  if (highlightLines) {
+    lineNumbers = highlightLines.split(',').map(v => v.split('-').map(v => parseInt(v, 10))) 
```

Comment:
Some comments here to describe what the input format should look like would be helpful! It would make maintaining this code easier in the future. 

Splitting up this line into several steps with well-named intermediate variables could help as well.

Reply:
Edited.